samples/ra-vpn/saml-multi-region/ca-certificate.yaml

```diff
@@ -0,0 +1,111 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mypkcs
+  namespace: sfcn-system
+stringData:
+  trustpoint: ssltp
+  password: test
+  # Change to your own CA Certificate
+  value: |
+    MIIS2gIBAzCCEqAGCSqGSIb3DQEHAaCCEpEEghKNMIISiTCCDQcGCSqGSIb3DQEH
```

Comment:
is this is a sample Cisco cert?

Reply:
I believe this is one of the internally used certificates for tests

Reply:
replaced it with dummy value

Reply:
replace the cert with dummy value for _samples/ra-vpn/route53-vpn/route53-vpn.yaml_ too